spec/unit/config_spec.rb

```diff
@@ -45,7 +45,7 @@
       ChefSpec::SoloRunner.new do |node|
         node.set['ohai']['disabled_plugins'] = [:passwd, "dmi"]
         node.set['chef_client']['config']['log_level'] = ":debug"
-        node.set['chef_client']['config']['ssl_verify_mode'] = ":verify_peer"
+        node.set['chef_client']['config']['ssl_verify_mode'] = ":verify_none"
```

Comment:
The context in the test is whether this attribute can be overridden -- you'll see that the default value is `:verify_peer` in the cookbook's attributes.